activity-container/build.gradle

```diff
@@ -1,5 +1,5 @@
 
 dependencies {
-    provided "com.google.dagger:dagger:2.12"
-    kapt 'com.google.dagger:dagger-compiler:2.12'
+    api "com.google.dagger:dagger:$daggerVersion"
```

Comment:
Shouldn't this be an `implementation` dependency? 🤔 
I'm still a little fuzzy on best practices, there.

Reply:
`implementation` doesn't leak out with this when you pull it into another project, `api` is the exact replacement for `compile`

Reply:
Yeah, I get what they do, I'm just not sure when to use which